ingest/muxer.go

```diff
@@ -634,11 +597,13 @@ mainLoop:
 				//we haven't hit our timeout yet, just continue
 				continue
 			}
-			//timeout, check state and force a return
-			//if we have a hot, filebacked cache, then endpoints are go for ingest
-			if im.cacheRunning && im.cacheError == nil && im.cacheFileBacked {
+
+			if im.cacheEnabled {
```

Comment:
check cache mode, not if it's enabled.

Reply:
Resolved by https://github.com/djfritz/gravwell/commit/34e537694c278b79fda2b717cd8b4a14df774a03